tests/s390x_tests/consoletest_TOOL_s390_vmconvert.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE’s openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+# Copyright (C) 2018 IBM Corp.
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary:  Based on consoletest_setup.pm (console test pre setup, stopping and disabling packagekit, install curl and tar to get logs and so on)
+# modified for running the testcase TOOL_s390_vmconvert on s390x.
+
+use base "s390base";
+use testapi;
+use utils;
+use strict;
+
+sub run {
+    my $self = shift;
+    $self->copy_testsuite('TOOL_s390_vmconvert');
+    
+    my $REPO_SUSE = get_var('REPO_SUSE');
```

Comment:
using `get_required_var` is better here, since it's more obvious to the user why it's failing

Reply:
Replaced with `get_required_var `